Jasmine/Common/ViewModels/LevelSelectorViewModel.swift

```diff
@@ -0,0 +1,94 @@
+import RealmSwift
+
+/// The ViewModel for the level selector part.
+class LevelSelectorViewModel: LevelSelectorViewModelProtocol {
+    /// The Levels object that manages the level objects
+    private var levels: Levels
+    /// Default levels as a level array
+    private var rawDefaultLevels: [Level] {
+        return levels.original
+    }
+    /// Custom levels as a level array
+    private var rawCustomLevels: [Level] {
+        return levels.custom
+    }
+    /// The defaults levels in the game.
+    var defaultLevels: [GameInfo] {
+        return rawDefaultLevels.map { level in
+            // TODO: Get the actual UUIDs
+            GameInfo(uuid: "A", levelName: level.name, gameType: level.gameType, gameMode: level.gameMode)
```

Comment:
Your UUID should be a legit UUID class, not a cooked up string, else there's no point.